### ISSUE TITLE
Ensure interactive controls are not nested

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/pulmap#1004]
### Errors in spec/features/accessibility_spec.rb

accessibility when visiting search results page complies with WCAG

```
nested-interactive: Ensure interactive controls are not nested (serious)
https://dequeuniversity.com/rules/axe/4.3/nested-interactive?application=axeAPI
The following 4 nodes violate this rule:
           
       Selector: #access
       HTML: <button class="btn btn-default btn-filter dropdown-toggle" type="button" id="access" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="true"><span><a data-no-turbolink="true" href="#">Access</a></span><span class="caret"></span></button>
       Fix any of the following:
       - Element has focusable descendants
               
       Selector: #institution
       HTML: <button class="btn btn-default btn-filter dropdown-toggle" type="button" id="institution" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="true"><span><a data-no-turbolink="true" href="#">Institution</a></span><span class="caret"></span></button>
       Fix any of the following:
       - Element has focusable descendants
               
       Selector: #format
       HTML: <button class="btn btn-default btn-filter dropdown-toggle" type="button" id="format" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="true"><span><a data-no-turbolink="true" href="#">Format</a></span><span class="caret"></span></button>
       Fix any of the following:
       - Element has focusable descendants
               
       Selector: #subject
       HTML: <button class="btn btn-default btn-filter dropdown-toggle" type="button" id="subject" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="true"><span><a data-no-turbolink="true" href="#">Subject</a></span><span class="caret"></span></button>
       Fix any of the following:
       - Element has focusable descendants
```

**Comment by tpendragon** [comment]
We're going to take a quick look. If it looks like it's going to be a lot of work to fix we'll deprioritize.

**Comment by thanyabegum** [comment]
This is something that would be best addressed upstream, or may be resolved after upgrading GeoBlacklight.
